Xamarin.Forms.Platform.iOS/Renderers/ShellPageRendererTracker.cs

```diff
@@ -478,6 +478,16 @@ protected virtual void OnSearchHandlerPropertyChanged(object sender, PropertyCha
 				UpdateSearchVisibility(_searchController);
 			else if (e.PropertyName == SearchHandler.IsSearchEnabledProperty.PropertyName)
 				UpdateSearchIsEnabled(_searchController);
+			else if (e.Is(SearchHandler.AutomationIdProperty))
+			{
+				UpdateAutomationId();
+			}
+		}
+
+		void UpdateAutomationId()
+		{
+			if (_searchHandler?.AutomationId != null && _searchController?.SearchBar != null)
+				_searchController.SearchBar.AccessibilityIdentifier = _searchHandler.AutomationId;
```

Comment:
Could `_searchController` be null here?

Reply:
Probably not but figured I'd just check for null anyway just to be safe

Reply:
nop because of  (_searchHandler?.AutomationId != null) before if _searchHandler is null it will not enter the if